src/containers/Output.js

```diff
@@ -113,6 +114,10 @@ class Output extends Component {
     }
   }
 
+  handleClickError(position) {
+    console.log(position)
```

Comment:
I expect to raise an event here somehow, but I don't know what pattern I should emulate here.